chainweb.cabal

```diff
@@ -357,9 +348,10 @@ library
         , Utils.Logging
         , Utils.Logging.Config
         , Utils.Logging.Trace
+        , Utils.PackageInfo
```

Comment:
Is there an away to encode the dependency on the cabal-plan tool in the cabal file? I vaguely remember that there's some stanza for tools.